Gemfile.lock

```diff
@@ -16,31 +16,42 @@ GEM
     colorator (1.1.0)
     commonmarker (0.17.13)
       ruby-enum (~> 0.5)
-    concurrent-ruby (1.1.7)
-    dnsruby (1.61.4)
+    concurrent-ruby (1.1.8)
+    dnsruby (1.61.5)
       simpleidn (~> 0.1)
     em-websocket (0.5.2)
       eventmachine (>= 0.12.9)
       http_parser.rb (~> 0.6.0)
-    ethon (0.12.0)
-      ffi (>= 1.3.0)
+    ethon (0.14.0)
+      ffi (>= 1.15.0)
     eventmachine (1.2.7)
     eventmachine (1.2.7-x64-mingw32)
-    execjs (2.7.0)
-    faraday (1.0.1)
+    execjs (2.8.1)
+    faraday (1.4.2)
+      faraday-em_http (~> 1.0)
+      faraday-em_synchrony (~> 1.0)
+      faraday-excon (~> 1.1)
+      faraday-net_http (~> 1.0)
+      faraday-net_http_persistent (~> 1.1)
       multipart-post (>= 1.2, < 3)
-    ffi (1.13.1)
-    ffi (1.13.1-x64-mingw32)
+      ruby2_keywords (>= 0.0.4)
+    faraday-em_http (1.0.0)
```

Comment:
the packages on lines 31-35 are listed again at lines 38-42, is this deliberate?

Reply:
I literally just ran `bundler update` 🤷

Reply:
fair enough, didn't want to aprrove & have things break